sockeye/arguments.py

```diff
@@ -1043,6 +1043,24 @@ def add_training_args(params):
                               nargs='*',
                               help="Manually specify names of parameters to fix during training. Default: %(default)s.")
 
+    # DeepSpeed arguments
+    train_params.add_argument('--local_rank',
+                               type=int_greater_or_equal(0),
+                               default=None,
+                               help='The DeepSpeed launcher (`deepspeed`) automatically adds this argument. When it is '
+                                    'present, training runs in DeepSpeed mode. This argument does not need to be '
+                                    'specified manually.')
+    train_params.add_argument('--deepspeed-fp16',
```

Comment:
are `--deepspeed-fp16` and `--deepspeed-bf16` mutually exclusive arguments? Would it make sense instead something like `--deepspeed-dtype {fp16,bf16}` to limit the number of arguments added to CLIs ?

Reply:
I agree that a single option would be easier to work with. Right now we mirror the option names in the [DeepSpeed config](https://www.deepspeed.ai/docs/config-json/#fp16-training-options) which uses separate sections for `fp16` and `bf16`. Enabling both is a valid DeepSpeed config, although DeepSpeed may raise its own error about it.